cpp/density/LocalDensity.cc

```diff
@@ -14,7 +14,7 @@ LocalDensity::LocalDensity(float r_max, float diameter)
     : m_box(box::Box()), m_r_max(r_max), m_diameter(diameter)
 {}
 
-LocalDensity::~LocalDensity() {}
+LocalDensity::~LocalDensity() = default;
```

Comment:
Since this PR seems to be mostly about cleaning source code, we should move this to the header

Reply:
That's a good call, I'll make that change. Please leave this conversation unresolved until then, thanks.